backend/src/models/__test__/user.spec.ts

```diff
@@ -0,0 +1,33 @@
+import { User } from '../user'
+import { TestDatabase } from './TestDatabase'
+import { clone } from '../../utils/clone'
+
+describe('Testing user component', () => {
+  const database = new TestDatabase()
+  const user = clone(new User())
+
+  user.email = 'x@email.com'
+  user.firstname = 'Vic'
+  user.lastname = 'Sidious'
+  user.password = 'pieTillIDie'
```

Comment:
This is for subsequent tests, for when you need to test a `POST` request perhaps, you will need a mock user.

Reply:
Okay